apps/cowswap-frontend/vite.config.mts

```diff
@@ -137,9 +137,17 @@ export default defineConfig(({ mode }) => {
 
     build: {
       assetsInlineLimit: 0, // prevent inlining assets
+      assetsDir: 'fonts', // serve fonts from /fonts/ directory
```

Comment:
https://swap-dev-git-fix-disable-workbox-verbose-logging-cowswap-dev.vercel.app/fonts/@safe-global-CvCgCxx7.js

Now everything in `/fonts`, even JS and CSS files

Reply:
let's call it `static`